x-pack/legacy/plugins/siem/public/components/paginated_table/index.test.tsx

```diff
@@ -384,9 +396,10 @@ describe('Paginated Table Component', () => {
       expect(updateActivePage.mock.calls[1][0]).toEqual(0);
     });
 
-    test('should call updateActivePage with 0 when an update prop changes', () => {
+    test('should update the page when the activePage is changed from redux', () => {
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const ourProps: BasicTableProps<any> = {
```

Comment:
Following the internet's mantra of typescript `any` is rarely needed due to the advancements of TypeScript, this line can be changed to:

```ts
const ourProps: BasicTableProps<unknown> = {
``` 

or

```ts
const ourProps: BasicTableProps<{}> = {
``` 

and the type checker will pass and the linter will go away. Food for thought for later on.

Reply:
I will fix it in my soon PR